x-pack/test/functional/services/cases/list.ts

```diff
@@ -192,7 +192,10 @@ export function CasesTableServiceProvider(
 
       const row = rows[index];
       await row.click();
-      await find.existsByCssSelector('[data-test-subj*="case-action-popover-"');
+      await retry.waitFor(
```

Comment:
I am not super familiar with these tests but I noticed that below `openAssigneesPopover` does `clickAndValidate`. I checked the implementation(in `test/functional/page_objects/common_page.ts`) and there is no `retry.waitFor`. Could this also be leading to some flaky tests?

Reply:
Actually I thought to use `clickAndValidate ` but we will have dynamic case id for `case-action-popover` so I didn't use. And we have already used `retry.waitFor` to wait for specific element in `changeStatus` and `changeSeverity`. So I used same logic